domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -352,6 +395,113 @@ class ExplorationProgressController @Inject constructor(
     }
   }
 
+  /**
+   *  Checks if checkpointing is enabled, if checkpointing is enabled this function creates a
```

Comment:
```suggestion
   * Checks if checkpointing is enabled, if checkpointing is enabled this function creates a
```

Spacing is wrong for the KDoc (should just be 1 space from the '*'). Please fix here & anywhere else in the PR.

Reply:
done

Reply:
done

Reply:
done